src/encoded/static/components/motifs.js

```diff
@@ -0,0 +1,130 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+// import * as logos from '../libs/d3-sequence-logo';
+
+function convertTextToObj(str) {
+    const cells = str.split('\n').map(el => el.split(/\s+/));
+    const obj = [];
+    cells.forEach((cell) => {
+        if (cell.length === 6) {
+            obj.push([+cell[1], +cell[2], +cell[3], +cell[4]]);
+        }
+    });
+    return obj;
+}
+
+export function getMotifData(pwmLink, fetch) {
+    return fetch(pwmLink, {
+        method: 'GET',
+        headers: {
+            Accept: 'application/json',
+        },
+    }).then((response) => {
+        if (response.ok) {
+            return response.text();
+        }
+        throw new Error('not ok');
+    }).catch((e) => {
+        console.log('OBJECT LOAD ERROR: %s', e);
+    });
+}
+
+// Display information on page as JSON formatted data
+class Motifs extends React.Component {
+    constructor(props, context) {
+        super(props, context);
```

Comment:
You’re not using `props` or context in this constructor, so you can just remove them.